app.py

```diff
@@ -152,6 +152,35 @@ def experience():
     return jsonify({})
 
 
+@app.route("/resume/experience/reorder", methods=["PUT"])
+def reorder_experience():
+    """
+    Reorder the experience entries based on the provided list of indices.
+    """
+    request_data = request.get_json()
+    if not request_data:
+        return jsonify({"error": "Request must be JSON"}), 400
+
+    if "order" not in request_data:
```

Comment:
We could use the existing method from helper.py here

Reply:
I don't think you need the first section because calling request.get_json() [will implicitly return status 400 if there is no JSON data.](https://flask.palletsprojects.com/en/3.0.x/api/#flask.Request.on_json_loading_failed)